config/yunikorn-history-server/config.yml

```diff
@@ -1,21 +1,22 @@
 yunikorn:
```

Comment:
`config/` now has a `yunikorn-history-server/` sub-directory - are we expecting to need other subdirectories underneath config?

Reply:
I don't know, are we, in any case we are prepared? 😄 

My personal preference for `config/` is the same as in `cmd/`